internal/models/source_model.go

```diff
@@ -0,0 +1,23 @@
+package models
+
+type Source struct {
+	// Required auto-generated PK.
+	ID uint64 `db:"id" json:"id"`
+
+	// Source ID in sources. Unique.
+	SourceID uint64 `db:"source_id" json:"source_id"`
```

Comment:
Do we need a `SourceID`, when we have the `AuthID`? In my mind I can query sources api just with the AuthID and it tells me the id of the source as well.
Is this to distinguish what provider this auth is for? If so, I'd rather used some internal identifier (e.g. [reuse what Lukas has](https://github.com/RHEnVision/provisioning-backend/blob/139ed1a25c67ae8d5d4a5e1da8460f25c445283c/internal/models/model_constants.go) ) and map it from sources

Reply:
from what I have checked the only thing needed is the auth id but Mikel from sources told me to store the source id as well. I will check it again with him.

Reply:
Trying out to sort our my understanding, so this is source id: https://github.com/RedHatInsights/sources-api-go/blob/main/model/source.go#L24

Reply:
checked it, we need the source id in order to delete it from sources. by deleting the authentication the source isn't deleted automatically.

Reply:
ok, let's keep the source, but can we map sourceID to provider type? can we distinguish between auth for azure and auth for amazon?